src/Lecture1.hs

```diff
@@ -30,6 +30,7 @@ module Lecture1
     , strSum
     , lowerAndGreater
     ) where
+import GHC.RTS.Flags (GCFlags(compact))
```

Comment:
I don't think you intended to import this module. I guess, VSCode automatically inserted it for you.

In any case, you can safely remove it 🙂

Reply:
Yes, I accidentally auto completed something and it got imported.